src/main/java/mekanism/common/tile/transmitter/TileEntityTransmitter.java

```diff
@@ -50,6 +52,10 @@ public void onWorldJoin()
 		{
 			TransmitterNetworkRegistry.registerOrphanTransmitter(getTransmitter());
 		}
+		else if(lastClientNetwork != null)
```

Comment:
This solved an issue that may be unique to Chisels and Bits. When previewing placement of a chiseled block it shows a ghost block that only affects the client side. This preview caused `onWorldSeparate` to be triggered on the client side which invalidated the transmitter network but if the block was not placed then no server change occurs and, thus, no sync between server and client. When the ghost block is moved away from the transmitter `onWorldJoin` is called on the client side which I used to correct this.

It's not game-breaking but since the client side thinks the transmitter has no network it appears as if nothing is going through the transmitter despite everything else updating correctly.